com.civicrm.hrjobroles/js/src/job-roles/filters/get-active-values.filter.js

```diff
@@ -1,13 +1,17 @@
+/* eslint-env amd */
+
 define([
-  'common/lodash',
-  'job-roles/filters/filters'
-], function(_,filters) {
+  'common/lodash'
+], function (_) {
   'use strict';
 
-  filters.filter('getActiveValues', ['$log', function($log) {
+  getActiveValues.__name = 'getActiveValues';
+  getActiveValues.$inject = ['$log'];
+
+  function getActiveValues ($log) {
```

Comment:
Since filters are functions that can be used somewhere else, it should be good to explain what the filter function does and why.